requirements.txt

```diff
@@ -19,3 +19,4 @@ django-axes==2.3.3
 shapely>=1.5.17
 markdown2
 alembic
+github+https://github.com/solar-c/django-bootstrap3.git
```

Comment:
Is it a fork? We should maybe host this code also on the OEP github account

Reply:
yes, its a fork. we could also fork it to the OEP github. but lets do it in a new branch/pull request.